docs/en/setup/service-agent/java-agent/configuration-discovery.md

```diff
@@ -0,0 +1,29 @@
+# CDS - Configuration Discovery Service
+
+CDS - Configuration Discovery Service provides the dynamic configuration for the agent, defined in [gRPC](https://github.com/apache/skywalking-data-collect-protocol/blob/master/language-agent/ConfigurationDiscoveryService.proto).
```

Comment:
In case of the link broken, we might commit sha instead of `master` in the proto link.

Reply:
I put this intentionally, as we basically don't move the proto files today. But if we link to the commitid, we have to upgrade this every time we changed. It is hard to update too.
We did a sync release tag in the protocol repo, and prefer reader could find the versions they need. 

https://github.com/apache/skywalking-data-collect-protocol/releases